dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProjectMapper.xml

```diff
@@ -82,20 +82,15 @@
     </select>
     <select id="queryProjectListPaging" resultType="org.apache.dolphinscheduler.dao.entity.Project">
         select
-        <include refid="baseSqlV2">
-            <property name="alias" value="p"/>
-        </include> ,
-        u.user_name as user_name,
-        count(distinct def.id) AS def_count,
-        count(distinct inst.id) as inst_running_count
+            <include refid="baseSqlV2">
+                <property name="alias" value="p"/>
+            </include> ,
+            u.user_name as user_name,
+            count(distinct def.id) AS def_count,
```

Comment:
Can we split this sql, don't use join here. We find use join is hard to maintain and is more easy to  produce slow query.

Reply:
I'll fix it.